cms/core/models/snippets.py

```diff
@@ -88,16 +88,7 @@ class GlossaryTerm(TranslatableMixin, PreviewableMixin, RevisionMixin, index.Ind
     ]
```

Comment:
question: what about making `ContactDetails` translatable?

Reply:
I later realised that snippets were out of scope for this ticket, but I made `ContactDetails` translatable in [4fb0dcc](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/4fb0dcc0339aab3df7ca8b9c5dea50000f821090)